frontend/src/app/features/book/components/book-browser/book-browser.component.html

```
@@ -1,496 +1,357 @@
                 </div>
-              </div>
-            </p-popover>
+              </p-popover>
+            </div>
+
+            <div class="sort-button-wrapper">
+              <button type="button" class="topbar-items topbar-item" (click)="sortPopover.toggle($event)"
+                (keydown.enter)="sortPopover.toggle($event)" [attr.aria-label]="t('tooltip.selectSorting')">
+                <i class="pi pi-sort" [pTooltip]="t('tooltip.selectSorting')" tooltipPosition="top">
+                </i>
+                @if (sortCriteriaCount() > 1) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)